lib/topology_lib_sflowtool/library.py

```diff
@@ -82,7 +84,7 @@ def stop(enode, state):
     state.sflowtool_pid = None
 
     return parse_sflowtool(
-        enode("cat {path}/sflowtool.log".format(path=path))
+        enode("cat {path}/sflowtool.log".format(path=path)), mode
```

Comment:
I think it is necessary just in stop function where you will indicate the mode that you used to capture (line or detail), for example:

collector_device.libs.sflowtool.start(mode='detail')
result = collector_device.libs.sflowtool.stop(mode='detail')